charts/sourcegraph/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Use `**BREAKING**:` to denote a breaking change
 
 ## Unreleased
 
+### Added
+
+- Added `allowedTopologies` support to storageclass [#188](https://github.com/sourcegraph/deploy-sourcegraph-helm/pull/188)
```

Comment:
Can you maybe explain in 1 sentence wha is the purpose? i.e. This is required to provision regional disks in GCP while having single node only - or something similar.

Reply:
I updated the text. would you take another look?